irohad/main/iroha_conf_literals.cpp

```diff
@@ -20,6 +20,7 @@ namespace config_members {
   const char *PgOpt = "pg_opt";
   const char *DbConfig = "database";
   const char *Host = "host";
+  const char *Ip = "ip";
```

Comment:
Please describe the new parameters at readthedocs or add an example of new parameters use (ip and utility_service)